src/components/builderPage/builderPage.jsx

```diff
@@ -45,22 +46,22 @@ class Builder extends Component {
         this.state = {
             layout: [],
             settings: {
-                name: "Character sheet",
+                name: 'Character sheet',
                 columns: 12,
                 rowHeight: 40,
-                size: "Letter",
+                size: 'letter',
                 height: null,
                 width: null,
             },
         };
     }
 
     componentDidMount() {
-        const savedLayout = localStorage.getItem("BuilderLayout");
+        const savedLayout = localStorage.getItem('axeBuilderLayout');
         if (savedLayout) {
             this.setState({ layout: JSON.parse(savedLayout) });
         }
-        const savedSettings = localStorage.getItem("sheetSettings");
+        const savedSettings = localStorage.getItem('axeSheetSettings');
         if (savedSettings) {
             console.table(JSON.parse(savedSettings));
```

Comment:
Looks like a leftover debugging line? It can always be removed later if it's still needed at this stage of development.

Reply:
Not needed, specially in this PR